<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>模式选择：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxMode">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>自动模式</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>手动模式</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>刷卡模式</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>维修模式</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>输入车位号：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditNum"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btnConnectDevice">
          <property name="text">
           <string>连接设备RTU</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnConfirm">
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnUp">
          <property name="text">
           <string>上升</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDown">
          <property name="text">
           <string>下降</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLeft">
          <property name="text">
           <string>左移</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRight">
          <property name="text">
           <string>右移</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnViewPlate">
          <property name="text">
           <string>查看扩展板</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnViewPlc">
          <property name="text">
           <string>查看PLCIO</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>模式选择</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>模式选择</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
